packages/core/src/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
this should not be hardcoded here, why do we need this?

Reply:
<html > should have lang? and default is en, if another language is loaded this is updated in i18next.js with the line: `document.documentElement.setAttribute('lang', lang);`

Reply:
resolved — the i18n package does indeed change the `lang` attribute of the `html` tag specifically when the language has been changed (which I thought it didn't do).